### PR TITLE
Remove AR limitations

### PR DESCRIPTION
**Pull request by Nicogp**
## Description
Changes included:
- [x] Change the FAQ  on how to create a custom script.
- [x] Rename old scripts(remove .sh).
- [x] Fix some logs
- [x] Deprecate <expect> in commands section

## Checks
- [x] It compiles without warnings.
- [ ] Spelling and grammar. 
- [ ] Used impersonal speech. 
- [ ] Used uppercase only on nouns.

### CODE REVIEW

File: source/deploying-with-puppet/wazuh-puppet-module/reference-wazuh-puppet/wazuh-manager-class.rst [hunk 1]
```diff
@@ -936,10 +936,7 @@ $command_name
   Human readable name for wazuh::activeresponse usage.
 
 $command_executable
-  Name of the executable. OSSEC comes preloaded with disable-account.sh, host-deny.sh, ipfw.sh, pf.sh, route-null.sh, firewall-drop.sh, ipfw_mac.sh, ossec-tweeter.sh, restart-ossec.sh.
-
-$command_expect
-  `Default srcip`
+  Name of the executable. WAZUH comes preloaded with disable-account, host-deny, ipfw, pf, route-null, firewall-drop, ipfw_mac, wazuh-slack, restart-wazuh.
```
Comment (TomasTurina) [inline source/deploying-with-puppet/wazuh-puppet-module/reference-wazuh-puppet/wazuh-manager-class.rst:939]: Remove `ipfw_mac`.

File: source/user-manual/capabilities/active-response/how-it-works.rst [hunk 2]
```diff
@@ -77,41 +76,39 @@ Wazuh is pre-configured with the following scripts for Linux:
...
+| firewalld-drop           | Adds an IP to the firewalld drop list                         |
 +--------------------------+---------------------------------------------------------------+
-| host-deny.sh             | Adds an IP to the /etc/hosts.deny file                        |
+| host-deny                | Adds an IP to the /etc/hosts.deny file                        |
 +--------------------------+---------------------------------------------------------------+
-| ip-customblock.sh        | Custom OSSEC block, easily modifiable for custom response     |
+| ip-customblock           | Custom OSSEC block, easily modifiable for custom response     |
 +--------------------------+---------------------------------------------------------------+
-| ipfw_mac.sh              | Firewall-drop response script created for the Mac OS          |
+| ipfw_mac                 | Firewall-drop response script created for the Mac OS          |
```
Comment (TomasTurina) [inline source/user-manual/capabilities/active-response/how-it-works.rst:89]: Remove this script.

File: source/user-manual/capabilities/active-response/remediation-configuration.rst [hunk 3]
```diff
@@ -18,35 +18,33 @@ Basic usage
...
+    <command>restart-wazuh</command>
     <location>local</location>
     <rules_id>10005</rules_id>
   </active-response>
 
 Windows automatic remediation
 -----------------------------
 
-In this example, the ``win_rout-null`` command is configured to use the ``route-null.cmd`` script using the data element ``srcip``.  The active response is configured to initiate the ``win_rout-null`` command on the local host when the rule has a higher alert level than 7.  This is a *Stateful* response with a timeout set at 900 seconds.
+In this example, the ``win_rout-null`` command is configured to use the ``route-null.exe`` script.  The active response is configured to initiate the ``win_rout-null`` command on the local host when the rule has a higher alert level than 7.  This is a *Stateful* response with a timeout set at 900 seconds.
```
Comment (TomasTurina) [inline source/user-manual/capabilities/active-response/remediation-configuration.rst:41]: Typo: `win_route-null`

**Review by TomasTurina — CHANGES_REQUESTED** [review]

File: source/deploying-with-puppet/wazuh-puppet-module/reference-wazuh-puppet/wazuh-manager-class.rst (hunk 1 above)
Comment (Nicogp) [reply source/deploying-with-puppet/wazuh-puppet-module/reference-wazuh-puppet/wazuh-manager-class.rst]: solved

File: source/user-manual/capabilities/active-response/how-it-works.rst (hunk 2 above)
Comment (Nicogp) [reply source/user-manual/capabilities/active-response/how-it-works.rst]: solved

File: source/user-manual/capabilities/active-response/remediation-configuration.rst (hunk 3 above)
Comment (Nicogp) [reply source/user-manual/capabilities/active-response/remediation-configuration.rst]: solved

File: source/user-manual/capabilities/active-response/remediation-configuration.rst [hunk 4]
```diff
@@ -112,14 +108,13 @@ Active response for a specified period of time
...
 The action of a stateful response continues for a specified period of time.
 
-In this example, the ``host-deny`` command is configured to use the ``host-deny.sh`` script using the data element ``srcip``.  The active response is configured to initiate the ``host-deny`` command on the local host when a rule with a higher alert level than 6 is fired.
+In this example, the ``host-deny`` command is configured to use the ``host-deny`` script.  The active response is configured to initiate the ``host-deny`` command on the local host when a rule with a higher alert level than 6 is fired.
 
 Command::
 
   <command>
     <name>host-deny</name>
     <executable>host-deny.sh</executable>
```
Comment (TomasTurina) [inline source/user-manual/capabilities/active-response/remediation-configuration.rst:117]: Remove `.sh`.

File: source/learning-wazuh/shellshock.rst [hunk 5]
```diff
@@ -234,8 +230,13 @@ this event 5 minutes apart.
...
+        2021/02/01 20:48:31 /var/ossec/active-response/bin/firewall-drop: {"version":1,"origin":{"name":"","module":"wazuh-execd"},"command":"delete","parameters":{"extra_args":[],"alert":{"timestamp":"2021-02-01T20:43:30.526+0000","rule":{"level":15,"description":"Shellshock attack detected","id":"31166","mitre":{"id":["T1068","T1190"],"tactic":["Privilege Escalation","Initial Access"],"technique":["Exploitation for Privilege Escalation","Exploit Public-Facing Application"]},"info":"CVE-2014-6271https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2014-6271","firedtimes":1,"mail":true,"groups":["web","accesslog","attack"],"pci_dss":["11.4"],"gdpr":["IV_35.7.d"],"nist_800_53":["SI.4"],"tsc":["CC6.1","CC6.8","CC7.2","CC7.3"]},"agent":{"id":"000","name":"ubuntu-bionic"},"manager":{"name":"ubuntu-bionic"},"id":"1612212210.6317002","full_log":"192.168.0.223 - - [01/Feb/2021:20:43:28 +0000] \"GET / HTTP/1.1\" 200 612 \"-\" \"() { :; }; /bin/cat /etc/passwd\"","decoder":{"name":"web-accesslog"},"data":{"protocol":"GET","srcip":"172.30.0.30","id":"200","url":"/"},"location":"/var/log/nginx/access.log"},"program":"/var/ossec/active-response/bin/firewall-drop"}}
```
Comment (TomasTurina) [inline source/learning-wazuh/shellshock.rst:237]: This log should be 5 minutes after the first one.

File: source/user-manual/capabilities/active-response/remediation-faq.rst [hunk 6]
```diff
@@ -11,22 +11,83 @@ FAQ
...
-.. code-block:: none
+.. code-block:: xml
 
-  <SCRIPT-NAME> <ACTION> <USER> <IP> <ALERT-ID> <RULE-ID> <AGENT> <FILENAME>
+  {
+    "version":1,
+    "origin":{
+        "name":"worker01",
+        "module":"wazuh-execd"
+        },
```
Comment (TomasTurina) [inline source/user-manual/capabilities/active-response/remediation-faq.rst:24]: Remove extra tab.

File: source/user-manual/capabilities/active-response/remediation-faq.rst [hunk 7]
```diff
@@ -11,22 +11,83 @@ FAQ
...
+Here is an example of the message that is passed to the ``firewall-drop`` AR:
+
+.. code-block:: json
+
+  {
+    "version":1,
+    "origin":{
+        "name":"worker01",
+        "module":"wazuh-execd"
+        },
```
Comment (TomasTurina) [inline source/user-manual/capabilities/active-response/remediation-faq.rst:44]: Remove extra tab.

File: source/user-manual/capabilities/active-response/remediation-faq.rst [hunk 8]
```diff
@@ -11,22 +11,83 @@ FAQ
...
+            "timestamp":"2021-02-01T20:58:44.830+0000",
+            "rule":{
+                "level":15,
+                "description":"Shellshock attack detected",
+                "id":"31168",
+                "mitre":{
+                    "id":["T1068","T1190"],
+                    "tactic":["Privilege Escalation","Initial Access"],
+                    "technique":["Exploitation for Privilege Escalation","Exploit Public-Facing Application"]
+                    },
```
Comment (TomasTurina) [inline source/user-manual/capabilities/active-response/remediation-faq.rst:58]: Remove extra tab.

File: source/user-manual/capabilities/active-response/remediation-faq.rst [hunk 9]
```diff
@@ -11,22 +11,83 @@ FAQ
...
+                "info":"CVE-2014-6271https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2014-6271",
+                "firedtimes":2,
+                "mail":true,
+                "groups":["web","accesslog","attack"],
+                "pci_dss":["11.4"],
+                "gdpr":["IV_35.7.d"],
+                "nist_800_53":["SI.4"],
+                "tsc":["CC6.1","CC6.8","CC7.2","CC7.3"]
+                },
```
Comment (TomasTurina) [inline source/user-manual/capabilities/active-response/remediation-faq.rst:67]: Remove extra tab.

**Review by TomasTurina — CHANGES_REQUESTED** [review]

**Review by TomasTurina — APPROVED** [review]

**Pull request merged**